test/test_stellar.py

```diff
@@ -56,7 +56,7 @@ def test_fetch_last_tx(self, m: requests_mock.Mocker):
             json=self.TRANSACTIONS,
         )
         last_tx = fetch_last_tx()
-        self.assertEqual(last_tx, datetime(2021, 5, 20))
```

Comment:
This actually has to stay the same as we are comparing against a hard coded reference (`2021-05-20T13:49:32Z`)

Reply:
Noted! Updating this :)